### ISSUE TITLE
[Feature request]: Check in individual assets in bookings

### ISSUE THREAD

**Issue by DroneProf** [Shelf-nu/shelf.nu#1374]
### Contact Details

rose196@purdue.edu

### Is your feature request related to a problem? Please describe?

Bookings often include multiple assets that are returned on different dates. The current solution requires the entire booking to be checked in and the remaining assets to be checked out again. 

### Describe the solution you'd like

In the same way that assets can be added to a booking before being checked out to the user, I would like to be able to check individual assets while retaining the custody ledger for all assets in the original booking.

**Comment by ej-mediacenter** [comment]
This is a feature that would also be very useful for my team. We often check out multiple assets to users who end up returning some of the items early and we would like to be able to circulate those items to other users immediately.

**Comment by carlosvirreira** [comment]
> This is a feature that would also be very useful for my team. We often check out multiple assets to users who end up returning some of the items early and we would like to be able to circulate those items to other users immediately.

Ok so the booking status would be still ongoing/overdue with some items not being returned?

**Comment by ej-mediacenter** [comment]
> > This is a feature that would also be very useful for my team. We often check out multiple assets to users who end up returning some of the items early and we would like to be able to circulate those items to other users immediately.
> 
> Ok so the booking status would be still ongoing/overdue with some items not being returned?

Yes. The checked in items would become available for other checkouts while the other items would remain checked out.

**Comment by DonKoko** [comment]
Closing this as we already allow this using the partial check-in feature.
